admin/class-smaily-for-wp-admin.php

```diff
@@ -73,15 +73,21 @@ public function enqueue_scripts() {
 	 * @since 3.0.0
 	 */
 	public function smaily_admin_render() {
-		global $wpdb;
-
 		// Create admin template.
 		$template = new Smaily_For_WP_Template( 'admin/partials/smaily-for-wp-admin-page.php' );
 
 		// Load configuration data.
-		$table_name = esc_sql( $wpdb->prefix . 'smaily_config' );
-		$data       = $wpdb->get_row( "SELECT * FROM `$table_name` LIMIT 1" );
-		$template->assign( (array) $data );
+		$api_credentials = get_option( 'smailyforwp_api_option' );
+		$form_options    = get_option( 'smailyforwp_form_option' );
+
+		$credentials_set = ! empty( $api_credentials['subdomain'] ) && ! empty( $api_credentials['username'] ) && ! empty( $api_credentials['password'] );
+
+		$template->assign(
+			array(
+				'form'                => isset( $form_options['form'] ) ? $form_options['form'] : '',
+				'api_credentials_set' => $credentials_set,
```

Comment:
When dealing with variables that can have only boolean values, it is good practice to use variable naming that has a yes/no answer.

For example with this variable it would be better to name it `has_credentials`, because `api_credentials_set` can be construed as a set/list/array of API credentials.

Reply:
Renamed it in 5eaf1ac